src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -937,4 +937,26 @@ public function hasErrors(): bool
     {
         return (bool) $this->errors;
     }
+
+    /**
+     * @internal
+     */
+    public function getFirstError(): ?string
```

Comment:
@nicolas-grekas This new internal method avoid to potentially compute several error messages, to finally only use the first one in all of our usage.